src/screens/Login/Login.tsx

```diff
@@ -92,14 +101,13 @@ function Login({ navigation }) {
           </View>
 
           <Input
-            label="Email"
+            label="Username"
             autoCapitalize="none"
-            keyboardType="email-address"
-            placeholder="Enter email"
-            setValue={value => handleChange({ email: value })}
-            icon={{ name: 'email', color: colors.primary }}
+            placeholder="Enter username"
+            setValue={value => handleChange({ username: value })}
+            icon={{ name: 'at', color: colors.primary }}
```

Comment:
does this icon name exists?

Reply:
Yes it does. It well render an `@` symbol icon